File: Week-03/1. Primitive/Solutions/soumya(GCS-1940064)/solution2.c
int convertFive(int n)
{
    int r,i=1,sum=0;
    while(n>0)
    {
        r=n%10;
        if(r==0)
        {sum=sum+(i*5);}
        else
        {sum=sum+(i*r);}
        n=n/10;
        i=i*10;
    }

    return sum;
}

File: Week-06/1. Primitive/Solutions/PrateekRai(CDE 1810256)/Solution-2.java
/*package whatever //do not write package name here */

import java.util.*;
import java.lang.*;
import java.io.*;

class GFG {
	public static void main (String[] args) {
		Scanner sc = new Scanner(System.in);
		int t = sc.nextInt();
		while(t>0)
		{
			t--;
			int n = sc.nextInt();
			int sum = 0;
			int a[] = new int[1000];
			for(int i=0,j=0;i<2000;i++)
			{
				if(i%2!=0)
				{
					a[j] = i;
					j++;
				}
			}
			for(int i=0;i<n;i++)
			{
				for(int j=0;j<=i;j++)
				{
					sum = sum+a[j];
				}
			}
			System.out.println(sum);
		}
	}
}
